### PR TITLE
further-layout-shifting-improvements

### PR DESCRIPTION
**Pull request by ntorbinskiy**

### CODE REVIEW

File: components/Post/LatestPosts.tsx [hunk 1]
```diff
@@ -17,11 +18,14 @@ const LatestPosts = ({ latestPosts }: Props) => {
...
+              <Image
                 src={post.featuredImage}
                 alt="blog post image"
-                width={102}
-                height={102}
+                layout="fill"
+                objectFit="cover"
+                width={800}
+                height={400}
+                quality={100}
```
Comment (1alexvash) [inline components/Post/LatestPosts.tsx:28]: @ntorbinskiy can you send a screenshot of a difference between default value, and this value?

File: components/Post/LatestPosts.tsx [hunk 2]
```diff
@@ -17,11 +18,14 @@ const LatestPosts = ({ latestPosts }: Props) => {
...
-              <img
+              <Image
                 src={post.featuredImage}
                 alt="blog post image"
-                width={102}
-                height={102}
+                layout="fill"
+                objectFit="cover"
+                width={800}
+                height={400}
```
Comment (1alexvash) [inline components/Post/LatestPosts.tsx:27]: @ntorbinskiy

Comment (1alexvash) [inline components/Post/LatestPosts.tsx:27]: @ntorbinskiy

File: components/PostCard.tsx [hunk 3]
```diff
@@ -16,7 +17,16 @@ const PostCard = ({ post }: Props) => {
         <div className="content">
           <Link href={`/post/${post.slug}`} as={undefined}>
             <a className="post-img">
-              <img src={post.featuredImage} alt="blog post image" />
+              {/* <img src={post.featuredImage} alt="blog post image" /> */}
```
Comment (1alexvash) [inline components/PostCard.tsx:20]: @ntorbinskiy remove

File: components/PostCard.tsx [hunk 4]
```diff
@@ -16,7 +17,16 @@ const PostCard = ({ post }: Props) => {
...
+              <Image
+                src={post.featuredImage}
+                alt="blog post image"
+                quality={100}
+                layout="fill"
+                objectFit="cover"
+                width={800}
+                height={400}
```
Comment (1alexvash) [inline components/PostCard.tsx:28]: @ntorbinskiy why those values in here?

Comment (ntorbinskiy) [reply components/PostCard.tsx]: @1alexvash 
They're nonsense here, deleted

File: components/Post/LatestPosts.tsx (hunk 1 above)
Comment (ntorbinskiy) [reply components/Post/LatestPosts.tsx:26]: image
But in this situation- I don't need it

Comment (ntorbinskiy) [reply components/Post/LatestPosts.tsx:26]: image
From [next.js docs](https://nextjs.org/docs/basic-features/image-optimization)
But in this situation

Comment (1alexvash) [reply components/Post/LatestPosts.tsx:26]: @ntorbinskiy why do you send me the documentation?

I asked a question and the image quality difference between the default value which is 75, and the value is 100 which you specified

File: components/PostCard.tsx [hunk 5]
```diff
@@ -16,7 +17,15 @@ const PostCard = ({ post }: Props) => {
...
             <a className="post-img">
-              <img src={post.featuredImage} alt="blog post image" />
+              <Image
+                src={post.featuredImage}
+                alt="blog post image"
+                layout="fill"
+                objectFit="cover"
+                quality={100}
+                // width={800}
+                // height={400}
```
Comment (1alexvash) [inline components/PostCard.tsx:27]:
```suggestion
```

File: styles/main.scss [hunk 6]
```diff
@@ -829,6 +829,7 @@ h2 {
   width: 102px;
   height: 102px;
   transition: all 0.3s ease-out;
+  position: relative;
```
Comment (1alexvash) [inline styles/main.scss:832]: @ntorbinskiy 
Um, does it do anything?

File: components/PostCard.tsx (hunk 5 above)
Comment (1alexvash) [inline components/PostCard.tsx:27]: @ntorbinskiy Why you are leaving those comments, they are here for historical reasons, or it should actually be some values?

File: components/Post/LatestPosts.tsx [hunk 7]
```diff
@@ -17,11 +18,14 @@ const LatestPosts = ({ latestPosts }: Props) => {
...
+              <Image
                 src={post.featuredImage}
                 alt="blog post image"
-                width={102}
-                height={102}
+                layout="fill"
+                objectFit="cover"
+                quality={100}
+                // width={800}
+                // height={400}
```
Comment (1alexvash) [inline components/Post/LatestPosts.tsx:28]: @ntorbinskiy same story as in the bottom comment

File: styles/main.scss (hunk 6 above)
Comment (ntorbinskiy) [reply styles/main.scss:832]: Yes, it does. In section Latest posts, Post will fill by his parent WITH position: relative. Without this line- it'll fill the whole screen. That is why I put it in here
image

File: components/PostCard.tsx (hunk 5 above)
Comment (ntorbinskiy) [reply components/PostCard.tsx:27]: No, I should delete them

File: styles/main.scss (hunk 6 above)
Comment (1alexvash) [reply styles/main.scss:832]: Um, okay, I just thought when you use Next Image it adds automatically

File: components/PostCard.tsx (hunk 5 above)
Comment (1alexvash) [reply components/PostCard.tsx:27]: @ntorbinskiy um okay, but for proper layout rendering without layout shifting, does it need any values?

Comment (ntorbinskiy) [reply components/PostCard.tsx:27]: image

Comment (alexander-vashchuk) [reply components/PostCard.tsx]: @ntorbinskiy Um okay, sounds cool, just don't forget to tag when sending a message on GitHub, as I do not receive email messages, and then it gets harder to find those messages

Comment (ntorbinskiy) [reply components/PostCard.tsx]: @alexander-vashchuk 
ok👍

**Pull request merged**